style.css

```diff
@@ -37,115 +41,120 @@ html {
+@media screen and (max-width: 600px) {
+	.navigation-links {
+	font-size: 2rem;
+	}
+}
+
+/*navigation*/
+
+/*welcome*/
+
+.welcome-banner {
+	background-color: #EEEEEE;
+	height: 90vh;
+	flex-direction: column;
 	font-family: 'Varela Round', sans-serif;
-	font-size: 6rem;
-	padding-top: 10%;
 	color: #000000;
-	/*deciding whether to keep element*/
-/*    -webkit-text-stroke-width: 0.5px;
-    -webkit-text-stroke-color: #EFB7B7;*/
 }
 
-.welcome-text { 
+@media screen and (max-width: 600px) {
+	.welcome-banner {
+	flex-direction: column;
+	}
+}
+
+.small-title {
+	font-size: 6rem;
 	display:flex;
 	align-items: center;
 	justify-content: center;
-	font-family: 'Varela Round', sans-serif;
-	font-size: 11rem;
-	background-color: #EEEEEE;
-	margin: 0;
-	color: #000000;
-/*deciding whether to keep element*/
-/*	-webkit-text-stroke-width: 0.5px;
-	-webkit-text-stroke-color: #EFB7B7;*/
+	padding-top: 10%;
+	margin-bottom: 0;
 }
 
-.about-me-title {
+@media screen and (max-width: 600px) {
+	.small-title {
+	font-size: 4rem;
+	}
+}
+
+.sparkle-inline {
+	width: 10% ;
+	display: inline;
+}
+
+.large-title {
+	font-size: 11rem;
+	margin: 0;
 	display:flex;
 	align-items: center;
 	justify-content: center;
-	padding-top: 0;
-	font-family: 'Varela Round', sans-serif;
-	font-size: 3rem;
 }
 
-.scallop-mid-section {
+@media screen and (max-width: 600px) {
+	.large-title {
+	font-size: 7rem;
+	}
+}
+
+/*come back to this once looked at mobile formatting*/
+
+/*welcome*/
+
+/*about me*/
+
+.about-section {
 	background-color: #BD4B4B;
 	padding: 50px;
 	height: 100vh;
-	position: relative;
-	
-}
-.about-me-text {
-	display: row;
+	display: flex;
 	align-items: center;
 	justify-content: center;
-	margin: auto;
-	width: 25%;
-	padding-top: 0;
+	gap: 50px;
+}
+
+.about-me-title {
+	padding-left: 30px;
+	font-family: 'Varela Round', sans-serif;
+	font-size: 3rem;
+}
+
+.about-me-text {
+	padding-left: 30px;
 	padding-bottom: 30px;
 	font-family: 'Varela Round', sans-serif;
 	font-size: 2rem;
 }
 
 .profile-photo {
-	width: 15%;
-	margin: auto;
-	border-radius: 25%;
-	display: flex;
+	width: 200px;
+	height: auto;
+	border-radius: 10%;
 }
 
-/*deciding whether I want to add scallops or not, so keep not to re-write*/
+/*about me*/
 
-/*.scallop-mid-section::before {
-  position:absolute;
-  top: -5px;
-  left: 0px;
-  width: 100%;
-  content:" ";
-  background:
-  radial-gradient(circle at 50% 0%, white 25%, #5496ff 26%, #5496ff 40%);
-  background-color: white ;
-  background-size:40px 100px;
-  height:50px;
-  background-repeat: repeat-x;
-}
-
-.scallop-mid-section::after {
-  position:absolute;
-/*  top: 50px;*/
-/*  left: 0px;
-  width: 100%;
-  content:" ";
-  background:
-  radial-gradient(circle at 50% 0%, #5496ff 25%, white 26%, white 40%);
-  background-color: #5496ff ;
-  background-size:40px 100px;
-  height:50px;
-  background-repeat: repeat-x;
-}*/*/
+/*portfolio*/
 
 .portfolio-section {
 	background-color: #EEEEEE;
+	padding: 50px;
 	height: 100vh;
```

Comment:
same issue here on a large screen with vh, a lot of white empty space below your projects